test/integration/default/build/vpc.tf

```diff
@@ -0,0 +1,33 @@
+#======================================================#
+#                    Flow Logs
+#======================================================#
+
+resource "aws_flow_log" "default_subnet" {
+  log_group_name = "${aws_cloudwatch_log_group.lmf_lg_3.name}"
+  iam_role_arn   = "${aws_iam_role.role_for_flow_log.arn}"
+  subnet_id      = "${aws_subnet.default.id}"
```

Comment:
When I run the integration tests, I currently get this error:
```
Error getting plugins: module root: 1 error(s) occurred:

* resource 'aws_flow_log.default_subnet' config: unknown resource 'aws_subnet.default' referenced in variable aws_subnet.default.id
rake aborted!
```
Try not to have your tests coupled.  Here, if you need a subnet, make a subnet just for testing the flow logs.  That way you are not dependent on anything else (which may get renamed, removed, disabled with a flag, etc).